### DISCUSSION TITLE
Please report bugs to Issues section (not Discussions)!

### DISCUSSION THREAD

**Discussion by yermak** [yermak/AudioBookConverter#174] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kruzzzr** [comment]
I've been using AudioBookConverter on and off for some time and I just ran into a problem. It won't convert. When I press "Start" nothing happens, not even an error message. I've tried everything I can think of. I doesn't convert any type of audio file I've tried. Does anyone have a suggestion for how I can resolve this problem?
